apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx

```diff
@@ -112,112 +102,117 @@ class StandardsEditor extends Component {
       }
     ];
     return columns;
-  }
+  };
 
-  handleRemoveStandardDialogOpen = standard => {
-    this.setState({standardToRemove: standard, confirmRemovalDialogOpen: true});
+  const handleRemoveStandardDialogOpen = standard => {
+    setStandardToRemove(standard);
+    setConfirmRemovalDialogOpen(true);
   };
 
-  handleRemoveStandardDialogClose = () => {
-    this.setState({standardToRemove: null, confirmRemovalDialogOpen: false});
+  const handleRemoveStandardDialogClose = () => {
+    setStandardToRemove(null);
+    setConfirmRemovalDialogOpen(false);
   };
 
-  removeStandard = () => {
-    this.props.removeStandard(
-      this.props.standardType,
-      this.state.standardToRemove
-    );
-    this.handleRemoveStandardDialogClose();
+  const removeStandard = () => {
```

Comment:
might be worth renaming this function, even though this works fine. we now have this function, the imported selector, and the removeStandard in props
previously it was disambiguated by "this." but the usage below isn't immediately clear.

Reply:
Thanks Tim, great catch!